macha/tests/test_macha.py

```diff
@@ -4,56 +4,68 @@
 import os
 import parmed as pm
 
-ligand_id = "../data/original/cdk2_32.pdb"
-parent_dir = "."
-original_dir = "macha/data/original"
+ligand_id = "cdk2_l32"
+parent_dir = "macha/data"
+original_dir = "original"
+
 
 def test_import():
     import parmed as pm
     import openbabel
 
+
 def test_checkInput():
 
-    proteinID,ligandID = checkInput(parent_dir=parent_dir, original_dir=original_dir, protein_name=None, input_ext="pdb")
+    protein_id, ligand_ids = checkInput(
+        parent_dir=parent_dir,
+        original_dir=original_dir,
+    )
+
+    # Assure that all ligands are found by the function
+    if not protein_id:
+        assert len(os.listdir(f"macha/data/{original_dir}")) == len(ligand_ids)
 
-    # assure that all ligands are found by the function
-    if not proteinID:
-        assert len(os.listdir(original_dir)) == len(ligandID)
 
 def test_createFolders():
 
-    preparation = Preparation(parent_dir=parent_dir, ligand_id="cdk2_l32",original_dir=original_dir,env="waterbox")
-    pdb_macha = preparation.pdb_file
+    preparation = Preparation(
+        parent_dir=parent_dir,
+        ligand_id=ligand_id,
+        original_dir=original_dir,
+        env="waterbox",
+        input_sanitation=False,
```

Comment:
This test is only passing with `input_sanitation=False` because it is a protein-ligand system. Is that intended? Because the default would be `input_sanitation=True` which will fail...

Reply:
This runs now with the new complexSeparator function, but I am looking at redoing some of the input strategy and harmonizing the complex functions.

Reply:
In other words, we shouldn't merge quite yet :-P